### ISSUE TITLE
The prediction accuracy for the masked token is ZERO when using the pretrained model. Does it make sense?

### ISSUE THREAD

**Issue by lazyfuzzypringle** [huggingface/pytorch-pretrained-BERT#641]
Hi, this question might be not relavent to the code but to BERT, I still hope I can find someone to help me out. 

I run the pretrained model BertForPreTraining and test it on my own text data. Because BERT has knowledge about language so I expect it to be able to predict the masked tokens with a reasonable accuracy, however it is zero. I run the model with model.eval() mode. Does it make sense that it predicts the masked tokens with zero? 

Thanks for the help.
